### ISSUE TITLE
508-defect-2 [HEADINGS, COGNITION]: Heading levels must only increase by one

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#24160]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Details

The **Choose a location** page has a blue alert box with an H3 heading. This should be an H2 because it's the first heading nested under the page H1. Screen shot attached below.

## Acceptance Criteria

- [ ] H3 is updated to an H2 in markup
- [ ] Visual styles remain consistent
- [ ] 0 axe-core violations thrown

## Environment

* https://staging.va.gov/health-care/schedule-view-va-appointments/appointments/new-covid-19-vaccine-booking/facility

## WCAG or Vendor Guidance (optional)

* https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html
* https://dequeuniversity.com/rules/axe/4.1/heading-order

## Screenshots or Trace Logs

![Screen Shot 2021-05-04 at 8 36 00 AM](https://user-images.githubusercontent.com/934879/117013598-a14a6f00-acb5-11eb-87ac-8e4c9ce0eb37.png)

**Comment by jbalboni** [comment]
This one is interesting, it didn't get caught by our Cypress tests because we don't test the variation with no home address in Cypress. When there's a home address, there's an h2 on this page and no skipped heading (it still makes sense to change the level of the box heading, though).

This makes me want to explore doing axe checks in our RTL tests, because Cypress tests aren't the best way to catch all visual variations of a page.
